src/content/blog/astro-520.mdx

```diff
@@ -0,0 +1,188 @@
+
+During development, Astro will display an error page instead of redirecting to help you catch any misconfigurations and bad links.
+
+To enable trailing slash redirects, set the `trailingSlash` option in your `astro.config.mjs`:
+
+```js title="astro.config.mjs" ins={3}
+export default defineConfig({
+	adapter: node({ mode: 'standalone' }),
+	trailingSlash: 'never', // or 'always'
+});
+```
+
+For more information, see the [`trailingSlash` documentation](https://docs.astro.build/en/reference/configuration-reference/#trailingslash).
+
+## External redirects
+
+Astro now supports defining external redirects to `http` or `https` destinations straight from the configuration. 
+
+Just like with internal redirects available since Astro 2.9, with an adapter you can also provide an object as the value. This allows you to specify a `status` code in addition to the new `destination`.
+
+```js
+import {defineConfig} from "astro/config"
+
+export default defineConfig({
+    redirects: {
+        "/about": "https://example.com/about",
+        "/news": {
+            status: 302,
+            destination: "https://example.com/news"
+        }
+    }
+})
+```
+
+See the updated [`redirects` configuration docs](https://docs.astro.build/en/reference/configuration-reference/#redirects) for more details.
+
+## TOML frontmatter in Markdown
+
+Astro's excellent support for Markdown and MDX has always included the YAML frontmatter, whether in file exports or content collections. With this release, Astro now also supports TOML frontmatter in both. This is useful for adding existing content files with TOML frontmatter to your project from another framework such as Hugo, or you may just prefer TOML over YAML!
+
+No configuration is required to use TOML frontmatter in your content files: just use `+++` to delimit the frontmatter and Astro will parse it as TOML.
+
+```md ins={1-6}
++++
+date = 2025-01-30
+title = 'Use TOML frontmatter in Astro!'
+[params]
+  author = 'Houston'
++++
+
+# Support for TOML frontmatter is here!
+```
+
+Thanks to [Colin Bate](https://colinbate.com/) who contributed this feature.
+
+## Experimental: `astro:config` 
+
+Astro allows the users to retrieve some configuration information in various ways: the use of the environment variable `import.meta.env`, the `Astro` global object in `.astro` files, and the `context` variable in middleware and endpoints.
+
+It could be hard to remember where the information was available: was it `import.meta.env.BASE` or `Astro.base`?
+
+With `astro:config`, we want to provide a *single* way to read the most useful configuration options from anywhere inside the project, with type-safety in mind! The end goal would be to deprecate the other ways of reading properties, and make `astro:config` the de facto standard. Please leave your feedback in the relevant [RFC](https://github.com/withastro/roadmap/pull/1106).
```

Comment:
```suggestion
Astro provides several ways to access your configuration settings from your project files: environment variables, the `Astro` global object in `.astro` files, and the `context` variable in middleware and endpoints. But, it can be hard to remember where the information is available: was it `import.meta.env.BASE` or `Astro.base`?

With `astro:config`, we want to provide a *single* way to read the most useful configuration options from anywhere inside your project. And, because it's Astro, we're doing it with type-safety in mind! 
```
Note, moving the part about future plans to the bottom (since it's future-looking ahead and asks for people to participate in the roadmap discussion) so they can get to / see the good stuff sooner!